website/docs/developing/ci-cd.md

```diff
@@ -1,9 +1,8 @@
 ﻿---
-id: ci-cd
-sidebar_position: 8
+sidebar_position: 45
 ---
 
-# Build and CI / CD pipeline
+# Build a CI / CD pipeline
```

Comment:
It was meant as dotnet Build and CI/CD pipeline and not "build a pipeline" but don't worry about it

Reply:
ok let me know if you want me to show you how to debug locally